kyaml/openapi/openapi.go

```diff
@@ -436,7 +436,8 @@ func initSchema() {
 		}
 
 		// parse the swagger, this should never fail
-		if err := parse(kubernetesapi.MustAsset(kubernetesAPIAssetName)); err != nil {
+		assetName := "kubernetesapi/" + kubernetesAPIDefaultVersion + "/swagger.json"
```

Comment:
nit:
```
assetName := filepath.Join(
   "kubernetesapi", kubernetesAPIDefaultVersion, "swagger.json")
```
this is friendlier to Windows.

Reply:
on that note, should the [kustomizationapi asset name](https://github.com/kubernetes-sigs/kustomize/blob/master/kyaml/openapi/openapi.go#L404) should be changed similarly?

Reply:
yes, this is just a general idiom to use when concatenating strings to make a filepath.
works better on windows, and also makes it easier to find places that create strings intended to hold filepaths
(we should make a type for that distinct from string, but its too late)

Reply:
oh wait, i just clicked on the link.  that's a const.  so no, cannot call a func there.